R/ahr_blinded.R

```diff
@@ -84,14 +84,14 @@ ahr_blinded <- function(
     hr = c(1, .6),
     ratio = 1) {
   # Input checking
-  if (!is.vector(hr, mode = "numeric") || min(hr) <= 0) {
-    stop("ahr_blinded: hr must be a vector of positive numbers.")
+  if (!is.numeric(hr) || min(hr) <= 0) {
+    stop("'hr' must be a vector of positive numbers.")
   }
-  if (!is.vector(intervals, mode = "numeric") || min(intervals) <= 0) {
-    stop("ahr_blinded: intervals must be a vector of positive numbers.")
```

Comment:
Hi Yihui, I have a question regarding the error message output. Typically, our practice in gsDesign2 is to print the function name at the beginning of the error message. This consideration arises from the possibility that the `ahr_blinded` function could potentially be invoked within another function. So... I am wondering if we should keep the `ahr_blinded` here?